tests/Router/RouterTest.php

```diff
@@ -156,6 +156,19 @@ public function testResolveUrl()
         $this->assertArrayHasKey('largecode', $params);
         $this->assertEquals('brown', $params['color']);
         $this->assertEquals('code/with/slashes/edit', $params['largecode']);
+
+        $rule = $router->reset()->route('testRuleId', '/color/:color/largecode/:largecode*|^[a-z]+\/[a-z]+$');
```

Comment:
@Flynsarmy Could you add another test like this one that tests a URL that would fail the validation explicitly (ie. it has numbers in it), and make sure that is correctly handled?

Reply:
https://github.com/octobercms/library/pull/463/files/1f5014bad4b7e52cb129f00159ec2da825c8419b#diff-da99d28fe49830941fd5098bd79edc52R169

Is this what you're looking for?

Reply:
@Flynsarmy nah, that one is just testing if there's an additional segment. I was hoping to have a test that covers a URL that fails the actual regular expression.

Reply:
Test added